### PR TITLE
Licence

### PR DESCRIPTION
**Pull request by CBjerke**
@DougManuel  and @StaceyFisher,  I've created a licence-terms.rmd for DemPoRT (based off the one for SCRiPS, Doug). Please review and once you are happy with it, we can pull it into the main branch.

### CODE REVIEW

**Comment by StaceyFisher** [comment]
Looks good to me. Thanks, @CBjerke.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
+The Dementia Population Risk Tool (DemPoRT) is a predictive algorithm for calculating 5-year risk for developing dementia. DemPoRT was developed population-level use and it is discriminating and well-calibrated across a wide range of population subgroups, despite using only self-reported risk factors (with a focus on health behaviours including smoking, diet and physical inactivity). DemPoRT uses include predicting how many people will develop dementia, calculate future dementia burden, identify high risk population subgroups, and inform the development of dementia prevention strategies. It can also be used by patients and their clinicians to assess an individual's risk of developing dementia.
 
 <ins>Stay tuned for individual and population-level DemPoRT implementation tools at www.projectbiglife.ca</ins>
```
Comment (DougManuel) [inline README.md:7]: Update to reflect published algorithm.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
 
 DemPoRT was developed and validated using Ontario individuals age 55+ from the 2001 to 2012 Canadian Community Health Surveys (CCHS). There were 75,460 survey respondents, 8,448 dementia events, and 348,677 person-years follow-up in the combined derivation and validation cohorts.
```
Comment (DougManuel) [inline README.md:9]: I a licence wouldn't normally include this level (performance of the application). Suggest removing but would reference the CMAJ paper.

**Review by DougManuel — COMMENTED** [review]
Looks good. Just suggest updating to reflect publication. 

Reading the licence, there is a lack of clarity about the licence to use the algorithm on the Project Big Life website (somewhat restricted), versus the licence to use the DemPoRT algorithm (largely unrestricted).

**Pull request merged**